docs/howto/krustlet-on-kubernetes-with-inlets.md

```diff
@@ -0,0 +1,278 @@
+# Running Krustlet on Kubernetes with inlets
+
+These are steps for running a krustlet node on your own computer. You can run any kind of 
+Kubernetes cluster you like, whether that's local on your computer or remote in managed 
+Kubernetes offering.
+
+These instructions are adapted from a [gist by Alex Ellis](https://gist.github.com/alexellis/d55d6d6a96ea9ae8d9d65b95297ec27e).
```

Comment:
If the instructions here are an adaptation from the original gist, why do we still need to provide the source?

Trying to suss out what's missing here in this guide that may be further explained in that gist.

Reply:
Is there a problem with giving credit? Can you think of another way that can be done (people don't look at commit history)

Reply:
No problem with giving credit. Just trying to determine why the link to the original gist is necessary. If you just want to name drop, name drop. :)